gcc/rust/Make-lang.in

```diff
@@ -355,7 +358,8 @@ RUST_INCLUDES = -I $(srcdir)/rust \
 	-I $(srcdir)/rust/checks/errors \
 	-I $(srcdir)/rust/checks/errors/privacy \
 	-I $(srcdir)/rust/util \
-        -I $(srcdir)/rust/metadata
+        -I $(srcdir)/rust/metadata \
+		-I $(srcdir)/../libgrust/libproc_macro
```

Comment:
I wonder if we want `../libgrust/libproc_macro` here or just `../libgrust`: My reasoning is, if we go for `-I $(srcdir)/../libgrust`, we won't have to touch this line again even if we create more libraries that both compiler and users will want to use, and it also makes it clearer at the include site that we're including "proc macro related" headers:

```c++
#include "libprocmacro/ident.h"
// vs
#include "ident.h"
```

Reply:
Very well, I changed it.